_sass/base/_variables.scss

```diff
@@ -17,7 +17,7 @@ $border-color: rgba(0, 0, 0, 0.1); // rgba recommended if using feature images
 
 // Typography colours
 $text-color: #383838;
-$link-color: $theme-color; // #1ABC9C;
```

Comment:
Actually. Does it make sense to change `$theme-color` to `#448CD6` instead??

Reply:
modified theme-color to #448CD6 and updated link-colour to theme-colour.